step-capstone/src/components/Unfinalized.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import TravelObject from './TravelObject';
+import { Grid } from '@material-ui/core';
+import TripSettingPopover from "./TripSettingPopover";
+
+function Header(props) {
+
+    function dropDownFunc(e) {
```

Comment:
like it has a form popover that we are using, material UI also has a Menu https://material-ui.com/components/menus/ - something to consider

Reply:
You're right. Let me change that

Reply:
Update: Since the other buttons are configured in raw CSS styles, now when I change the drop down button using Material UI, it costs so much time to regain the styles. Thus, I consider not to change it.

Reply:
I would say it should at least be a separate PR, up to you all whether you think the change is worth the effort period.